cpu/native/mtd/mtd_native.c

```diff
@@ -103,6 +105,41 @@ static int _write(mtd_dev_t *dev, const void *buff, uint32_t addr, uint32_t size
     return 0;
 }
 
+static int _write_page(mtd_dev_t *dev, const void *buff, uint32_t page, uint32_t offset,
+                       uint32_t size)
+{
+    mtd_native_dev_t *_dev = (mtd_native_dev_t*) dev;
+    uint32_t addr = page * dev->page_size + offset;
+
+    DEBUG("mtd_native: write from page %" PRIx32 ", offset 0x%" PRIx32 " count %" PRIu32 "\n",
+          page, offset, size);
+
+    if (page > dev->sector_count * dev->pages_per_sector) {
```

Comment:
Why not just call `_write()` here?

Reply:
I'd like to get rid of `_write()` eventually